server/src/controllers/appointments.controller.js

```diff
@@ -42,17 +34,7 @@ export async function newAppointment(req, res, next) {
       });
     }
 
-    const geocodeData = await geocodeAddress(address);
-
-    if (!geocodeData.results.length) {
-      res.status(404);
-      if (geocodeData.status === "ZERO_RESULTS") {
-        return next({ message: "The address submitted is not valid." });
-      }
-      return next({
-        message: "Unable to locate the address. Please verify and try again.",
-      });
-    }
+    const { lat, long } = getLatLong(address);
```

Comment:
Need to define `getLatLong` as its currently throwing a reference error

Reply:
This also throwing a SqlLite error: `no such column LONG`. I think its LON?
<img width="1078" alt="Screenshot 2024-12-05 at 9 08 01 PM" src="https://github.com/user-attachments/assets/a53efec3-b6e5-48a6-8417-a40d854e3d19">

Reply:
<img width="410" alt="Screenshot 2024-12-05 at 9 10 00 PM" src="https://github.com/user-attachments/assets/0829b214-e769-4638-a576-f1e66a8c0d66">